presto-kafka07/src/main/java/com/facebook/presto/kafka/KafkaConnectorConfig.java

```diff
@@ -76,7 +76,17 @@
     /**
      * Fetch size
      */
-    private int fetchSize = 10 * 1024 * 1024;
+    private DataSize fetchSize = new DataSize(10, Unit.MEGABYTE);
+
+    /**
+     * Default Query Interval
```

Comment:
I think based on the code style in Presto, it's better just assign the default value here. I agree that the variable name here is a little bit confusing. But the "defaultQueryInterval" means that this value is the default query interval if the query interval is not specified in SQL.